lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -272,13 +282,64 @@ static LogicalResult verifyCellOp(CellOp cell) {
 //===----------------------------------------------------------------------===//
 static LogicalResult verifyAssignOp(AssignOp assign) {
   auto parent = assign->getParentOp();
-  if (!(isa<GroupOp>(parent) || isa<WiresOp>(parent)))
+  if (!(isa<GroupOp, WiresOp>(parent)))
```

Comment:
Plz drop the redundant parens around the isa call

Reply:
Hey @lattner this is still in draft. Did I accidentally request review from you?

Reply:
Fixed in either case :-)

Reply:
No, just saw it fly by.